bootstrap/090_post_setup.sql

```diff
@@ -286,6 +286,39 @@ exception
         RAISE;
 END;
 
+CREATE OR REPLACE TASK TASKS.WAREHOUSE_LOAD_MAINTENANCE
+    SCHEDULE = '60 minute'
+    ALLOW_OVERLAPPING_EXECUTION = FALSE
+    USER_TASK_MANAGED_INITIAL_WAREHOUSE_SIZE = "XSMALL"
+    AS
+BEGIN
+    let dt timestamp := current_timestamp();
+    let wh resultset := (select name from internal.sfwarehouses);
+    let wh_cur cursor for wh;
+    for wh_row in wh_cur do
+        let wh_name varchar := wh_row.name;
+        let input variant := (select output from INTERNAL.TASK_WAREHOUSE_LOAD_EVENTS where success and warehouse_name = :wh_name order by run desc limit 1);
+        begin
+            let stmt resultset := (call internal.refresh_one_warehouse_load_history(:wh_name));
+            let stmt_cur cursor for stmt;
+            let counter number := 0;
+            for stmt_row in stmt_cur do
+                execute immediate stmt_row.sql;
+                let new_count number := (select * from TABLE(RESULT_SCAN(LAST_QUERY_ID())));
```

Comment:
we're collecting the number of rows inserted into our materialized table from the table function? A comment would help make this clear (through the ~3 layers of indirection) or maybe just calling this somethign like `rows_inserted_for_wh` instead of the generic `count`

Reply:
fixed